yarn.lock

```diff
@@ -3,143 +3,157 @@
+    babel-plugin-typescript-to-proptypes "^1.2.1"
 
 "@airbnb/config-eslint@^2.1.3":
-  version "2.5.1"
-  resolved "https://registry.yarnpkg.com/@airbnb/config-eslint/-/config-eslint-2.5.1.tgz#c4d58c53bcdc5ba3982b136265872e7392b7439e"
-  integrity sha512-k8rToPOUwoG30uEenbKrE/bg7QCxlIEtVEbPFtZZuCQ0x8M5zQrbeRrsSA69FCUbnicm8vtQ+odti+Yd4kQqeA==
-  dependencies:
-    "@airbnb/nimbus-common" "^2.1.1"
-    "@beemo/core" "^1.1.1"
-    "@beemo/driver-eslint" "^1.2.0"
-    "@typescript-eslint/eslint-plugin" "^2.17.0"
-    "@typescript-eslint/parser" "^2.17.0"
+  version "2.6.0"
+  resolved "https://registry.yarnpkg.com/@airbnb/config-eslint/-/config-eslint-2.6.0.tgz#4e9d6cedf8cc2ebe553ad616c17f27ee502ec579"
+  integrity sha512-dLjs2rr97gS06heH1sLceoQGB3U5RLZ5p4oO+CBzAcWwCrv7QwIDsbNIbnJWuj6unb+CY91818h+cX2GdbAshg==
+  dependencies:
+    "@airbnb/nimbus-common" "^2.1.3"
+    "@beemo/core" "^1.1.4"
+    "@beemo/driver-eslint" "^1.2.2"
+    "@typescript-eslint/eslint-plugin" "^2.20.0"
+    "@typescript-eslint/parser" "^2.20.0"
     babel-eslint "^10.0.3"
     eslint "^6.8.0"
     eslint-config-airbnb "^18.0.1"
-    eslint-config-prettier "^6.9.0"
-    eslint-plugin-import "^2.20.0"
-    eslint-plugin-jest "^23.6.0"
+    eslint-config-prettier "^6.10.0"
+    eslint-plugin-import "^2.20.1"
+    eslint-plugin-jest "^23.7.0"
     eslint-plugin-jsx-a11y "^6.2.3"
     eslint-plugin-prettier "^3.1.2"
     eslint-plugin-promise "^4.2.1"
-    eslint-plugin-react "^7.18.0"
-    eslint-plugin-react-hooks "^2.3.0"
-    eslint-plugin-unicorn "^15.0.1"
+    eslint-plugin-react "^7.18.3"
+    eslint-plugin-react-hooks "^2.4.0"
+    eslint-plugin-unicorn "^16.1.1"
 
 "@airbnb/config-jest@^2.1.3":
-  version "2.2.2"
-  resolved "https://registry.yarnpkg.com/@airbnb/config-jest/-/config-jest-2.2.2.tgz#2d50245376e67c91e865121e523655046fdc03e7"
-  integrity sha512-Th63Swbe21AnHLb08Lz74uaSuy+4weydLOzNQuyv0SWaCtedJGMB5jdIadCwUMg80uNnpVQSuGnIRH/2TfvmqA==
-  dependencies:
-    "@airbnb/nimbus-common" "^2.1.1"
-    "@babel/core" "^7.8.3"
-    "@beemo/core" "^1.1.1"
-    "@beemo/driver-babel" "^1.2.0"
-    "@beemo/driver-jest" "^1.1.1"
+  version "2.2.4"
+  resolved "https://registry.yarnpkg.com/@airbnb/config-jest/-/config-jest-2.2.4.tgz#ed68c745b048af36994134538fc82ef57f318469"
+  integrity sha512-UOmUDBmvN2nFUe7LrrbXo3+OHiYW5utnkbqrtFiq5RS9kZhbDZ2Ihe74pYvpZISKaZid5iEJJDcOnibs9KA18Q==
+  dependencies:
+    "@airbnb/nimbus-common" "^2.1.3"
+    "@babel/core" "^7.8.4"
+    "@beemo/core" "^1.1.4"
+    "@beemo/driver-babel" "^1.2.2"
+    "@beemo/driver-jest" "^1.2.1"
     airbnb-js-shims "^2.2.1"
     dom-storage "^2.1.0"
     jest "^25.1.0"
 
 "@airbnb/config-prettier@^2.0.4":
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/@airbnb/config-prettier/-/config-prettier-2.1.1.tgz#cf18b45555f92309b470634da176ddc9ebb53e81"
-  integrity sha512-lt/q40KUWK3GdQEARGbJG5aCoPaFmlLCIY/9C/0dIYsCT/MmwGBQ6c5ez7JZ5dvWaGZb5Z0eNUnz44+LzHjq/Q==
+  version "2.1.3"
```

Comment:
I'm surprised that a minor version change resulted in the `'` => `"` changes 🤔 do you think that's right?

also just saw that there are `3.0` versions of all of these, should probably upgrade sometime 🔜 .

Reply:
this is actually my fault. I had the Deno prettier config set as my default formatter on save 🤦